src/sign/hashedTypes.json

```diff
@@ -55,5 +55,9 @@
   "daaf0ff7d3e2ec189cf851228c386c7a4c432076d0ed197864916dd3f656531c": "subscribe",
   "f542edb6f9fe79d0b85554667991352be2fcca85df3dd3c6a6f451d189c7c25d": "unsubscribe",
   "6f6cdd15a1e9e6c4ee544231c4fa7b6c7e5183bc2b37fa4bd1a695f458348ab7": "delete-space",
-  "1aad7825b991457fca04aae48a2a49d815ada524e10316af0da5522ea48b3df6": "alias"
+  "1aad7825b991457fca04aae48a2a49d815ada524e10316af0da5522ea48b3df6": "alias",
+  "8ea9074bff3a30cb61f4f0f0f142c9335c6be828feba0571a45de3f1fd0319c0": "alias",
```

Comment:
These new types are to support signatures using starknet? How to generate these? 

Could you create a script file similar to https://github.com/snapshot-labs/snapshot.js/blob/master/scripts%2FgenerateHashWithTypes.ts

Reply:
These new types are using `string` instead of `address` for the `from` fields.

Reply:
So from requirements, we are missing `statement` here 

![Untitled 4](https://github.com/snapshot-labs/snapshot.js/assets/15967809/2865c5b8-8739-4765-b284-265d22d589d7)


also maybe we could add `subscribe`, `unsubcribe` also?

Reply:
these will be added in future?

Reply:
`subscribe` and `unsubscribe` are not used in sx.

We could add `statement`, but will not be able to test it, it's not merged yet on sx